src/renderer/services/chain/fees/unstake.ts

```diff
@@ -6,74 +6,64 @@ import * as Rx from 'rxjs'
 import * as RxOp from 'rxjs/operators'
 
 import { BASE_CHAIN } from '../../../const'
-import { isBaseChain } from '../../../helpers/chainHelper'
-import { liveData, LiveData } from '../../../helpers/rx/liveData'
-import { observableState } from '../../../helpers/stateHelper'
+import { LiveData } from '../../../helpers/rx/liveData'
+import { triggerStream } from '../../../helpers/stateHelper'
 import * as BNB from '../../binance/service'
-import * as BTC from '../../bitcoin/context'
 import { selectedPoolChain$ } from '../../midgard/common'
-import { getUnstakeMemo$ } from '../memo'
-import { FeeLD, StakeFeesLD } from '../types'
+import { FeeLD, UnstakeFeeLD } from '../types'
 import { reloadStakeFeesByChain } from './fees.helper'
 
-const { get$: unstakePercent$, set: updateUnstakePercent } = observableState(0)
+// const { get$: unstakePercent$, set: updateUnstakePercent } = observableState(0)
 
-// reload fees
-const updateUnstakeFeesEffect$ = Rx.combineLatest([selectedPoolChain$, unstakePercent$]).pipe(
+// `TriggerStream` to reload unstake fees
+const { stream$: reloadUnstakeFees$, trigger: reloadUnstakeFees } = triggerStream()
+
+/**
+ * reload fees
+ *
+ * Has to be used ONLY on an appopriate screen
+ * @example
+ * useSubscription(updateUnstakeFeesEffect$) - ONLY
+ */
```

Comment:
[ntipick] Same typo: `appropriate view` (just a missing `r`). Last - ONLY can be removed I think.